src/Database/Schema/SqliteSchemaDialect.php

```diff
@@ -52,6 +52,10 @@ class SqliteSchemaDialect extends SchemaDialect
      */
     protected function _convertColumn(string $column): array
     {
+        if (empty($column)) {
+            return ['type' => TableSchema::TYPE_STRING, 'length' => null];
```

Comment:
The fallback type at the end of this function is `return ['type' => TableSchema::TYPE_TEXT, 'length' => null];` so maybe we should use the same for consistency?